doc/sphinx-guides/source/installation/config.rst

```diff
@@ -617,6 +617,15 @@ this provider.
 - :ref:`dataverse.pid.ezid.username`
 - :ref:`dataverse.pid.ezid.password`
 
+**JVM Options for CrossRef:**
```

Comment:
The documentation, and some of the code, is for 'legacy' single pid configuration but I think we could/should only support handling the new multipid way of doing config.  Although I think you've added it, I think it's reasonable to not support the legacy style at all for cross ref (since the idea of legacy was to support people who had already deployed with the old single provider jvm options and no one could have used CrossRef before now (except the developer's group).

To change the docs - see the DataCite section in the guide and add the info there (probably right after DataCite/before Handle.

Reply:
will modify the doc